### PR TITLE
Travis checks for README files

### PR DESCRIPTION
**Pull request by IATkachenko**
To check changes in README files by travis-ci added set of scripts to Tests/travis and updated travis configuration.

All checks goes in own check phase.

Fixes: #268

### CODE REVIEW

File: Tests/travis/README_tabs.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  . "$functions"
+else
+  echo "functions file '$functions' not found!"
+  exit 1
+fi
+
+e=0
+while read f; do
+  [ -z "$f" ] && continue
+  r=`getAddedLines "$f" | grep -P '^.*?\t+.*'`;
```
Comment (mgolosova) [inline Tests/travis/README_tabs.sh]: Won\`t `grep -P '\t'` work just as well?

File: Tests/travis/README_trailing_spaces.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  . "$functions"
+else
+  echo "functions file '$functions' not found!"
+  exit 1
+fi
+
+e=0
+while read f; do
+  [ -z "$f" ] && continue
+  r=`getAddedLines "$f" | grep -e '^.*\s\+$'`;
```
Comment (mgolosova) [inline Tests/travis/README_trailing_spaces.sh]: Won\`t `grep -e '\s$'` work?
I mean, single trailing space must already be enough to say that the line is corrupted.

File: Tests/travis/README_tabs.sh (hunk 1 above)
Comment (IATkachenko) [reply Tests/travis/README_tabs.sh]: Thank you, regexp was simplified.

File: Tests/travis/README_trailing_spaces.sh (hunk 2 above)
Comment (IATkachenko) [reply Tests/travis/README_trailing_spaces.sh]: Thank you, regexp was simplified.

File: Tests/travis/functions.sh [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  git diff HEAD^ -- "$file" | grep -v "^+++ b/$file" | grep -e '^+' | sed -e 's/^+//'
+}
+
+getLinesFromFile() {
+  file="$1"
+  lines="$2"
+
+  echo "$lines" | sort -u | while IFS= read -r l; do
+      grep -n "^$l\$" -- "$file"
+  done
```
Comment (mgolosova) [inline Tests/travis/functions.sh]: `sort -u` is to be sure that every line will be found only once, right?
Won\`t it make `grep` output out of order? Maybe sort and make unique the result (line numbers), not lines themselves?

*(Didn\`t try it yet, so just a thought)*

Comment (IATkachenko) [reply Tests/travis/functions.sh]: If we don't make line unique, then grep will do work twice.
I'll add ordering by line number into output.

**Comment by IATkachenko** [comment]
Commit logs will be reworded.

**Pull request merged**